examples/bri-3/src/bri/zeroKnowledgeProof/models/document.ts

```diff
@@ -0,0 +1,13 @@
+import { AutoMap } from '@automapper/classes';
+export class DocumentObject {
+  @AutoMap()
+  documentObjectType: string;
```

Comment:
what is documentObjectType? can it be an enum?

what is documentObjectInput?

Could you please definitions of this fields to the proposal document which is accompanying the work on this PR.

Reply:
These fields were suggested by Andreas to accommodate future implementations which can handle different types of documents. `documentObject` is made up of `documentObjectType` as string, and `documentObjectInput` as `object which must conform to the schema defined for the `documentObjectType``. Should we keep it in this PR or remove it and include it in future PR when it is required?